src/main/sensors/acceleration.c

```diff
@@ -424,9 +424,9 @@ static void performAcclerationCalibration(rollAndPitchTrims_t *rollAndPitchTrims
 
     if (isOnFinalAccelerationCalibrationCycle()) {
         // Calculate average, shift Z down by acc_1G and store values in EEPROM at end of calibration
-        accelerationTrims->raw[X] = (a[X] + (CALIBRATING_ACC_CYCLES / 2)) / CALIBRATING_ACC_CYCLES;
-        accelerationTrims->raw[Y] = (a[Y] + (CALIBRATING_ACC_CYCLES / 2)) / CALIBRATING_ACC_CYCLES;
-        accelerationTrims->raw[Z] = (a[Z] + (CALIBRATING_ACC_CYCLES / 2)) / CALIBRATING_ACC_CYCLES - acc.dev.acc_1G;
+        accelerationTrims->raw[X] = lrintf((a[X] + (CALIBRATING_ACC_CYCLES / 2)) / CALIBRATING_ACC_CYCLES);
```

Comment:
Old code did round almost correctly (floor(x + .5) == round(x), but integer division is `trunc()`, so it works only for positive values), now it's offset by +0.5
Also both methods are IMO identical (except for negative value) - accADC is integer, so sum is exact and  division will produce identical result

Reply:
sounds like a good case for a unit test.